packages/runtime/test-runtime-utils/src/generateToken.ts

```diff
@@ -0,0 +1,56 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import type { ITokenClaims, IUser, ScopeType } from "@fluidframework/protocol-definitions";
+import { KJUR as jsrsasign } from "jsrsasign";
+import { v4 as uuid } from "uuid";
+
+/**
```

Comment:
Should we mark the duplicate code so we can dedup it later?  or this is the long term solution?

Reply:
I'll add a clarifying comment and follow-up issue. I'm open to suggestions about better long-term options. I don't love the duplication long term, but it seems to be such "dumb" code that doing complex acrobatics to avoid duplication also seems wrong.